app/src/main/res/values/strings.xml

```diff
@@ -196,13 +197,13 @@
     <string name="warning_read_only">Grant KeePass DX write access to your SD card to save changes to your database.</string>
     <string name="warning_unmounted">Mount your SD card to create or load your database.</string>
     <string name="warning_empty_password">Do you really want to use an empty string as your password?</string>
-    <string name="warning_no_encryption_key">Are you sure you do not want to use any encryption key?</string>
+    <string name="warning_no_encryption_key">Are you sure you do not want to encrypt your database?</string>
```

Comment:
Not the same meaning, do not use encryption key does not mean that the database will not be encrypted, it is always encrypted.